### DISCUSSION TITLE
Comparison with react-beautiful-dnd

### DISCUSSION THREAD

**Discussion by headfire94** [clauderic/dnd-kit#481] [Q&A]
You mentioned that your lib doesn't use html5 API, but react-beautiful-dnd is also doesn't use it.
It also has almost zero deps (small utils), customizable and performance based on description.
What was the motivation to create new library is not clear from readme

**Comment by Brian-McBride** [comment]
Seeing that [react-beautiful-dnd](https://github.com/atlassian/react-beautiful-dnd) is no longer maintained, I'd say that is a solid top motivational reason for us all to contribute to `dnd-kit`. I'll be using it on my next project. 

But even before Altassian decided to abandon their lib, here are some of the reasons I was going to pick up `dnd-kit` on a project to give it a go. My motivation was mainly around the use of Typescript over Flow and far more code development activity.

**Comment by zhang0ZGC** [comment]
react-beautiful-dnd does not seems to support grid sorting😥.
But for simple sorting, react-beautiful-dnd seems to be easier🤣.

**Comment by xHomu** [comment]
While we wait for Clauderic to chime in, he mentioned his thoughts on this briefly in the original announcement post :

> claudericd 317d
> I would preface my answer by saying I haven't used react-beautiful-dnd extensively; the main difference would be that react-beautiful-dnd is specifically built for lists, while @dndkit supports a much wider range of use cases, such as grids, trees, 2D games, and much more. I would say it's more similar to react-dnd than to react-beautiful-dnd.

https://www.reddit.com/r/reactjs/comments/kr3qhx/httpsdndkitcom_a_lightweight_performant/

I found this post particularly illuminating on the design decisions behind dnd kit (and a big motivation for why I build with this repo) https://www.reddit.com/r/reactjs/comments/kr3qhx/comment/gi988v7/
